src/components/Wizard/Inputs/FormControlBlock.tsx

```diff
@@ -0,0 +1,27 @@
+import { Block } from "baseui/block";
+import { FormControl, FormControlProps } from "baseui/form-control";
+
+export const FormControlBlock = ({
+  title,
+  help,
+  error,
+  children,
+}: FormControlBlockProps) => (
+  <Block
+    display="flex"
+    alignItems="flex-start"
+    justifyContent="flex-start"
+    flexDirection="column"
+  >
+    <FormControl label={title} caption={help} error={error}>
+      {children}
+    </FormControl>
+  </Block>
+);
+
+export type FormControlBlockProps = {
+  title?: FormControlProps["label"];
+  help?: FormControlProps["caption"];
+  children: React.ReactNode;
```

Comment:
Import `React` depending on the answer to my bringing into scope question in the `FormStep.tsx` file.

Reply:
Updated